src/amber/router/pipe/flash.cr

```diff
@@ -106,6 +106,15 @@ module Amber
           self
         end
 
+        def merge!(other : Hash(String, String)?)
```

Comment:
Is there any reason to allow nil for the input value? It seems as if merge!(nil) should just exception and not compile.